ibm_vpd_utils.hpp

```diff
@@ -360,5 +360,11 @@ void setBusProperty(const std::string& service, const std::string& object,
  */
 std::variant<int64_t, std::string>
     readBIOSAttribute(const std::string& attrName);
+
+/**
+ * @brief Returns the power state for chassis0
```

Comment:
chassis0 ?

Reply:
Is there a concern here, @jinuthomas ? I don't follow the coment